dotcom-rendering/src/components/LiveblogRightMultipleAdSlots.tsx

```diff
@@ -0,0 +1,112 @@
+import { css } from '@emotion/react';
+import { useEffect, useState } from 'react';
+import { isServer } from '../lib/isServer';
+import {
+	AD_CONTAINER_HEIGHT,
+	PADDING_BOTTOM,
+	SPACE_BETWEEN_ADS,
+} from '../lib/liveblog-right-ad-constants';
+import { AdSlot } from './AdSlot';
+
+const rightColumn: HTMLElement | null = !isServer
+	? window.document.querySelector('[data-gu-name="right-column"]')
+	: null;
```

Comment:
Is there a reason that this selection is done at the top level of the script, and not within the effect itself? I guess if the right column is SSRed it might make no difference, but if you inlined it you'd be able to avoid the `isServer` check.

Reply:
Yep, good point. I had copied the pattern from another file but you're right, there's no need to be defined outside of the `useEffect`